CHANGELOG.rst

```diff
@@ -6,6 +6,70 @@
 Synapse Changelog
 *****************
 
+v2.153.0 - TBD
+=====================
+
+Model Changes
+-------------
+- Update to the ``inet`` and ``ou`` models.
+
+  (`#3393 <https://github.com/vertexproject/synapse/pull/3393>`_)
+  (`#3396 <https://github.com/vertexproject/synapse/pull/3396>`_)
+
+  **Deprecated Properties**
+
+  ``inet:web:acct``
+    The ``inet:web:acct`` form had the following properties marked as deprecated:
+
+    * ``name:en``
+    * ``realname:en``
+
+  ``inet:web:group``
+    The ``inet:web:group`` form had the following property marked as deprecated:
+
+    * ``name:en``
+
+  ``ou:industry``
+    The ``ou:industry`` form had the following property marked as deprecated:
+
+    * ``subs``
+
+Features and Enhancements
+-------------------------
+- Add a new Storm API, ``$lib.iters.zip()``, to iterate over sequences of
+  items together.
+  (`#3392 <https://github.com/vertexproject/synapse/pull/3392>`_)
+  (`#3398 <https://github.com/vertexproject/synapse/pull/3398>`_)
+- Add a Storm command ``stats.countby`` to tally occurrences of values and
+  display a barchart representing the values.
+  (`#3385 <https://github.com/vertexproject/synapse/pull/3385>`_)
+- Update the Storm command ``auth.user.mod`` to allow setting a user as admin
+  on a specific auth gate.
+  (`#3391 <https://github.com/vertexproject/synapse/pull/3391>`_)
+- Add an ``errors`` parameter to ``$lib.axon.readlines()``,
+  ``$lib.axon.csvrows()``, and ``$lib.axon.jsonlines()``. This parameter
+  defaults to ``ignore`` to ignore any decoding errors that are encountered
+  when decoding text.
+  (`#3395 <https://github.com/vertexproject/synapse/pull/3395>`_)
+- Lower the maximum allowed version of the ``pyopenssl`` library.
+  (`#3399 <https://github.com/vertexproject/synapse/pull/3399>`_)
+
+Bugfixes
+--------
+- Fix a bug in the ``Cortex.syncLayersEvents()`` and
+  ``Cortex.syncIndexEvents()`` APIs which caused layers to stop sending their
+  node edits under certain conditions.
+  (`#3394 <https://github.com/vertexproject/synapse/pull/3394>`_)
+- Storm now raises as ``BadSyntaxError`` when attempting to filter by wildcard
+  tags or tagprops when a value is specified for the filter.
+  (`#3373 <https://github.com/vertexproject/synapse/pull/3373>`_)
+
+Improved Documentation
+----------------------
+- Update the documentation for the Storm ``view.exec`` command to explain the
```

Comment:
This went out in 2.152.0: https://github.com/vertexproject/synapse/releases/tag/v2.152.0